bazelrio/scripts/deploy/Deploy.java

```diff
@@ -85,13 +94,20 @@ public static void main(String[] args) throws IOException {
         client.addHostKeyVerifier(new NoopKeyVerifier());
         client.useCompression();
 
-        establishSession(client, parsedArgs.get("team_number"), verbose);
+        if (!establishSession(client, parsedArgs.get("team_number"), verbose)) {
+            System.err.println("Couldn't find a roboRIO");
```

Comment:
Why continue? Should this throw / return early? I feel like nothing else will work right?